builds/org.eclipse.n4js.product.build/N4JS__IDE.launch

```diff
@@ -66,6 +59,8 @@
 <setEntry value="org.eclipse.wst.xml_core.feature:default"/>
 <setEntry value="org.eclipse.wst.xml_ui.feature:default"/>
 <setEntry value="org.eclipse.wst.xml_userdoc.feature:default"/>
+<setEntry value="org.eclipse.epp.mpc:default"/>
```

Comment:
Why did you remove `org.eclipse.epp.mpc` bundle from the run configuration? This bundle allows market place. So theoretically, we can try the market place in the IDE started from within the development IDE. Practically, due to some other issues, this didn't work for me but still.

Reply:
According to my tests, this configuration allows for the use of the marketplace in a smith-launched IDE.

Reply:
Exactly, that's why it may be a good idea to keep it there?

Reply:
Ah it is kept. The diff view is a bit confusing.